### ISSUE TITLE
count: not working correctly/consistently  

### ISSUE THREAD

**Issue by raprasad** [opendifferentialprivacy/opendp-ux#446]
Needs more troubleshooting

- Via the UI, the application "Count" stat is not always working consistently 
   - Validation passes, but actual execution run fails
   - `run_chain` error message from OpenDP lib: 
      -  `"inferred type is i32, expected f64. See https://github.com/opendp/opendp/discussions/298 (Exception)"`
- The failing UI params do work (both Float and Categorical) when doing a `StatSpec` level test run
  - This tests uses the OpenDP library directly, sans UI. (These UI params and related file have been added to the unit tests)
- To do:
   - Track differences in input between validation and execution (if any)
   - Basically some debugging to do...

**Comment by raprasad** [comment]
(currently not able to reproduce, needs further checking, moving back to "Up Next")

**Comment by raprasad** [comment]
fixed
